deploy/packer/app/scripts/deploy.sh

```diff
@@ -33,3 +33,7 @@ sudo npm install -g babel@4.7.16
 sudo sh -c 'echo "v4l2loopback" >> /etc/modules'
 sudo sh -c 'echo "snd-aloop" >> /etc/modules'
 sudo depmod
+
+# install busybox
+sudo curl -o /worker/bin-utils/busybox http://www.busybox.net/downloads/binaries/latest/busybox-x86_64
```

Comment:
So, I see that this will be done on the ec2 instances as well as the changes made for the vagrant environment, but this file needs to be present in our ci image I would suspect